DeleteContact/index.ts

```diff
@@ -1,37 +1,16 @@
 import { AzureFunction, Context, HttpRequest } from "@azure/functions";
-import { CosmosClient } from "@azure/cosmos";
-
-const endpoint = "https://lionel-onboarding.documents.azure.com:443/";
-const key =
-  "9sjqsze0dBFWOwxF0eZ2sfaJJbULXDkXsTJRvDL17mqKP4YiMXtMNI3JDszCWfBNlwiXaoSElnzHACDbOBFQJg==";
-const client = new CosmosClient({ endpoint, key });
-
-async function main() {
-  // The rest of the README samples are designed to be pasted into this function body
-}
-
-main().catch((error) => {
-  console.error(error);
-});
+import { contactsRepository } from "../repository/contactsRepository";
 
 const httpTrigger: AzureFunction = async function (
   context: Context,
   req: HttpRequest
 ): Promise<void> {
-  const { database } = await client.databases.createIfNotExists({
-    id: "onboarding",
-  });
-  console.log(database.id);
-  const { container } = await database.containers.createIfNotExists({
-    id: "contacts",
-  });
-  console.log(container.id);
+  
+  const contactDao = new contactsRepository();
+  await contactDao.init();
 
   const contactId = context.bindingData.id;
   const clientId = context.bindingData.clientId;
-  console.log("contactId: ", contactId);
-
-  await container.item(contactId, clientId).delete();
+  contactDao.deleteContact(contactId, clientId);
```

Comment:
no await for this command, how will you know it was successful?
What is the correct response code for this operation?